FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -17,18 +18,21 @@ final class AttachmentManager: NSObject {
     private weak var controller: UIViewController?
     private let filesManager: FilesManagerType
 
-    init(controller: UIViewController,
-         filesManager: FilesManagerType) {
+    init(
+        controller: UIViewController,
+        filesManager: FilesManagerType
+    ) {
         self.controller = controller
         self.filesManager = filesManager
     }
 
     private func showFileSharedAlert(with url: URL) {
+        guard let view = controller?.view else { return }
         let alert = UIAlertController(
             title: "message_attachment_saved_successfully_title".localized,
             message: "message_attachment_saved_successfully_message".localized,
             preferredStyle: .alert
-        )
+        ).popoverPresentation(style: .centred(view))
```

Comment:
There is no need for `popoverPresentation` modifier for `UIAlertController` with `preferredStyle: .alert`, it's needed only for `.actionSheet`.
Alerts are automatically centred in the parent view.

Reply:
@sosnovsky please check issue and attached screenshot.